README.md

```diff
@@ -21,35 +21,36 @@ Given that different groups of genes both contribute to these traits of interest
+| [2_correlation_analysis](2_correlation_analysis) | This folder contains analysis notebooks to detect gene co-expression modules starting with gene expression data, applying Pearson correlation and then clustering on this correlation matrix to obtain gene modules.|
+| [3_common_genes_analysis](3_common_genes_analysis) | This folder contains analysis notebooks to compare common DEGs found in [prior work](https://github.com/greenelab/generic-expression-patterns/blob/master/pseudomonas_analysis/2_identify_generic_genes_pathways.ipynb) to core and accessory genes|
+| [4_acc_acc_analysis](4_acc_acc_analysis) | This folder contains analysis notebooks to examine accessory-accessory gene modules.|
+| [5_core_core_analysis](5_core_core_analysis) | This folder contains analysis notebooks to examine the stability of core genes across strains.|
+| [6_core_acc_analysis](6_core_acc_analysis) | This folder contains analysis notebooks to examine the relationship between core genes and accessory genes.|
+| [scripts](scripts) | This folder contains supporting functions that other notebooks in this repository will use.|
 | [data](data) | This folder contains metadata used for different analyses.|
-| [explore_data](explore_data) | This folder contains analysis notebooks to visualize the expression data to get a sense for the variation contained.|
-| [processing](processing) | This folder contains analysis notebooks to determine what threshold to use to partition the gene expression data into PAO1 and PA14 compendia.|
 
 
 ## Usage
-If you want to run notebooks from core-accessory-interactome, you need to follow these steps to create a conda environment and download the necessary data.
+*Operating Systems:* Mac OS, Linux (Note: bioconda libraries not available in Windows)
 
-1. In the terminal execute the following steps to setup the conda environment:
-```bash
-# conda version 4.6.12
-conda env create -f environment.yml
-conda activate core_acc
-pip install -e .
-```
-
-2. Update paths in `core_acc_modules/paths.py`
+In order to run this simulation on your own gene expression data the following steps should be performed:
 
-3. Download reference transcriptomes from Pseudomonas.com and store in appropriate path defined in `core_acc_modules/paths.py`
-
-4. In the terminal launch jupyter notebook within your environment:
+First you need to set up your local repository:
+1. Download and install [github's large file tracker](https://git-lfs.github.com/).
+2. Install [miniconda](https://docs.conda.io/en/latest/miniconda.html)
+3. Clone the `core-accessory-interactome` repository by running the following command in the terminal:
+```
+git clone https://github.com/greenelab/core-accessory-interactome.git
+```
+Note: Git automatically detects the LFS-tracked files and clones them via http.
+4. Navigate into cloned repo by running the following command in the terminal:
```

Comment:
This change and the next one are pedantic, I'm sorry. It's just that points 4 and 5 read differently than the other points since you drop the 'the'
```suggestion
4. Navigate into the cloned repo by running the following command in the terminal:
```